open-weather-connector-test/src_test/com/axonivy/connector/openweather/test/it/AirPollutionProcessTest.java

```diff
@@ -0,0 +1,98 @@
+package com.axonivy.connector.openweather.test.it;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.time.Duration;
+import java.time.OffsetDateTime;
+
+import org.apache.http.HttpStatus;
+import org.junit.jupiter.api.Test;
+import org.openweathermap.api.data2_5.client.AirPollution;
+
+import ch.ivyteam.ivy.bpm.engine.client.BpmClient;
+import ch.ivyteam.ivy.bpm.engine.client.ExecutionResult;
+import ch.ivyteam.ivy.bpm.error.BpmError;
+
+public class AirPollutionProcessTest extends BaseProcessTest {
```

Comment:
seems as if you have duplicated all tests now .... just to run it also towards a real environment?
this seems very hard to maintain to me; shouldn't we better instrument junit5 to run the same tests towards multiple environments? this can be achieved with TestTemplates...
![junit5-templates](https://github.com/user-attachments/assets/ce5af679-ba4c-4c1f-89e2-86628066e701)